modular_skyrat/modules/antilag/code/admin_procs.dm

```diff
@@ -0,0 +1,67 @@
+/client/proc/kick_afk()
+	set category = "Admin"
+	set name = "Kick ALL AFK Clients"
+	set desc = "Kicks all clients who are AFK, both ingame and in the lobby."
+
+	var/kick_type_choice = tgui_alert("Do you want to kick all AFK clients, or only those observing or in the lobby?", "All of them", "Only those in the lobby/observing", "Cancel")
+
+	if(!kick_type_choice || kick_type_choice == "Cancel")
+		return
+
+	var/list/kicked_client_names = list()
+
+	if(kick_type_choice == "All of them")
+		for(var/client/iterating_client as anything in GLOB.clients)
+			if(!iterating_client)
```

Comment:
What is the point in this

Reply:
Clients can be deleted at any moment and that list is liable to have a null in it.